app/src/main/java/com/ucm/informatica/spread/Presenter/MapFragmentPresenter.java

```diff
@@ -153,34 +121,6 @@ public void onSwitchLocationMode() {
         mapFragmentView.renderLocationView(currentMode);
     }
 
-
```

Comment:
i didnt change it, just move it, first constructors, then public methods and finally private methods